modules/subnet/subnet.tf

```diff
@@ -10,6 +10,8 @@ resource "oci_core_subnet" "vcn_subnet" {
   cidr_block     = each.value.cidr_block
   compartment_id = var.compartment_id
   vcn_id         = var.vcn_id
+  availability_domain = lookup(each.value, "availability_domain", null)
+
```

Comment:
please format your code. commented code should be removed unless there's a specific reason for it

Reply:
Yes , I will remove the commented code